products/amazon-neptune.md

```diff
@@ -36,9 +36,9 @@ releases:
 
 -   releaseCycle: "1.3.1.0"
     releaseDate: 2024-06-06
-    upgradeVersion: "1.3.2.0"
+    upgradeVersion: "1.3.1.0"
```

Comment:
The upgrade path as per https://docs.aws.amazon.com/neptune/latest/userguide/engine-releases.html is 1.3.2.0

Reply:
Thanks a lot for putting this in evidence @captn3m0